README.md

```diff
@@ -147,18 +147,37 @@ Thank you to the following individuals for contributing to this release:
 * Mauricio Vásquez
 * Yusuke Tsutsumi
 
+The [v0.5 beta
+release](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v0.5.0) release includes:
+
+- W3C Correlation Context Propagation
+- OpenTelemetry Collector Exporter Integration for both metrics and traces
+- Metrics SDK
+- Global configuration module
+- Documentation improvements
+
+Thank you to the following individuals for contributing to this release:
+
+* Alex Boten
```

Comment:
I'm wondering if we should keep doing this or just add an AUTHORS file and add everybody that has contributed to the project there.